src/http_controller/inotification_long_polling_controller.ts

```diff
@@ -0,0 +1,215 @@
+import {HttpRequestWithIdentity} from '@atlas-engine/http_contracts';
+
+import {Response} from 'express';
+
+/**
+ * Describes a HTTPController for managing HttpRequests related to Notifications.
+ */
+export interface INotificationLongPollingController {
+  /*
+  * Waits for the CronjobCreated notification and returns the result
```

Comment:
Wie bei den User Domain Contracts, würd ich die Headlines auch eher so formulieren:

```suggestion
  * Waits for a CronjobCreated notification and resolves with the notification's content.
```